File: benchmark/python/task6_fileio.py
from collections import Counter

def count_words(input_file, output_file):
    with open(input_file) as f:
        words = f.read().split()
    counts = Counter(words)
    with open(output_file, 'w') as f:
        for word, count in counts.items():
            f.write(f"{word}: {count}\n")

File: benchmark/lua/task6_fileio.lua
function count_words(input_file, output_file)
    local counts = {}
    local file = io.open(input_file, "r")
    local text = file:read("*all")
    file:close()

    for word in text:gmatch("%S+") do
        counts[word] = (counts[word] or 0) + 1
    end

    file = io.open(output_file, "w")
    for word, count in pairs(counts) do
        file:write(word .. ": " .. count .. "\n")
    end
    file:close()
end
